VetoModules/makeVetoList.py

```diff
@@ -92,16 +110,25 @@
 
 iLine=0
 for iLine in range(len(pixelCfgMapLines)):
+    print iLine,pixelCfgMapLines[iLine]
     mapLine=pixelCfgMapLines[iLine]
     if mapLine.find("key ")!=-1:
         key=mapLine.split()[1]
         if key in globalConfigKeys:
-            #print pixelCfgMapLines[iLine+1]
-            if pixelCfgMapLines[iLine+1].find("detconfig")!=-1:
-                thisdetconfig=pixelCfgMapLines[iLine+1].split()[1]
-                #print thisdetconfig
-                if thisdetconfig not in detConfigs:
-                    detConfigs.append(thisdetconfig)
+            iLoop=1
```

Comment:
This could be a for loop though.

Reply:
I made it like this so that if they change the format (line order) it would still work.  In 2017 they did change that.